### PR TITLE
exporters: adding VM resolver

### PR DESCRIPTION
**Pull request by KalmanMeth**
Per recommendation of @hunchback

### CODE REVIEW

File: contrib/exporters/secadvisor/mod/resolve_vm.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+// NewResolveVm creates a new name resolver
+func NewResolveVm(cfg *viper.Viper) Resolver {
+	gremlinClient := client.NewGremlinQueryHelper(core.CfgAuthOpts(cfg))
+
+	return &resolveVm{
+		gremlinClient: gremlinClient,
+	}
+}
+
+type resolveVm struct {
```
Comment (hunchback) [inline contrib/exporters/secadvisor/mod/resolve_vm.go]: rename to `resolveVM`

File: contrib/exporters/secadvisor/mod/resolve_vm.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+// IPToName resolve ip to name
+func (r *resolveVm) IPToName(ipString, nodeTID string) (string, error) {
+	node, err := r.gremlinClient.GetNode(g.G.V().Has("RoutingTables.Src", ipString))
+	if err != nil {
+		return "", err
+	}
+
+	if node == nil {
```
Comment (hunchback) [inline contrib/exporters/secadvisor/mod/resolve_vm.go]: redundant - `r.gremlinClient.GetNode()` handles this

File: contrib/exporters/secadvisor/mod/resolve_vm.go [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+	"github.com/spf13/viper"
+
+	"github.com/skydive-project/skydive/api/client"
+	"github.com/skydive-project/skydive/common"
+	"github.com/skydive-project/skydive/contrib/exporters/core"
+	g "github.com/skydive-project/skydive/gremlin"
+)
+
+// NewResolveVm creates a new name resolver
+func NewResolveVm(cfg *viper.Viper) Resolver {
```
[the remark this answers is not in the document]
Comment (KalmanMeth) [reply contrib/exporters/secadvisor/mod/resolve_vm.go]: OK

File: contrib/exporters/secadvisor/mod/resolve_vm.go (hunk 1 above)
Comment (KalmanMeth) [reply contrib/exporters/secadvisor/mod/resolve_vm.go]: OK

File: contrib/exporters/secadvisor/mod/resolve_vm.go (hunk 2 above)
Comment (KalmanMeth) [reply contrib/exporters/secadvisor/mod/resolve_vm.go]: OK

**Comment by safchain** [comment]
run tests

**Comment by safchain** [comment]
run tests

**Comment by hunchback** [comment]
@KalmanMeth - please fix tests

**Comment by KalmanMeth** [comment]
run skydive-compile-tests

**Comment by KalmanMeth** [comment]
run skydive-functional-tests-backend-orientdb
run skydive-functional-tests-backend-elasticsearch
run skydive-cdd-overview-tests

**Comment by hunchback** [comment]
run skydive-cdd-overview-tests

**Pull request merged**